src/shared/components/annotation/OncoKB.tsx

```diff
@@ -78,7 +78,15 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
             <span className={`${annotationStyles["annotation-item"]}`} />
         );
 
-        if (this.props.indicator)
+        if (this.props.indicator === null) {
+            // null means there is an error...
+            oncoKbContent = this.errorIcon();
+        }
+        else if (this.props.indicator === undefined) {
+            // undefined means still loading...
```

Comment:
it would be good, I think, to have a prop `loading`. This is a visualization, so `loading` and `error` as props, visualizations specs, makes sense to me, rather than overloading a prop whose name (indicator) doesn't indicate relation to `loading` or `error` viz.